patches/server/0624-Allow-for-Component-suggestion-tooltips-in-AsyncTabC.patch

```diff
@@ -35,10 +35,11 @@ index e3d0c7f357a579811de1b55d8fde018aa55fea47..acbaba8750c373b90f6af0fb592f7226
 -            completions.forEach(builder::suggest);
 +            final com.mojang.brigadier.suggestion.SuggestionsBuilder builder = builder0.createOffset(builder0.getInput().lastIndexOf(' ') + 1);
 +            completions.forEach(completion -> {
-+                if (completion.tooltip() == null) {
-+                    builder.suggest(completion.suggestion());
++                final Integer intSuggestion = com.google.common.primitives.Ints.tryParse(completion.suggestion());
```

Comment:
would prefer a local variable for the result of `completion.tooltip() == null ? null : PaperAdventure.asVanilla(completion.tooltip())`

Reply:
Looking back at it it seems like `PaperAdventure.asVanilla(n.k.a.t.Component)` returns null if the argument is null. I was wondering if you'd still prefer the tooltip component in a local var or just call the function directly when calling `builder.suggest`?

Reply:
calling the function directly is fine